### PR TITLE
[DEVC-961] Replace ntpd monit call with init script call [master]

### PR DESCRIPTION
**Pull request by benjaminaltieri**
Last remnant of `monit`

### CODE REVIEW

File: package/piksi_system_daemon/overlay/etc/init.d/update_ntp_config [hunk 1]
```diff
@@ -4,4 +4,5 @@ ntp_cfg="$1"; shift
 
 rm /etc/ntp.conf
 ln -sf "$ntp_cfg" /etc/ntp.conf
-monit restart ntpd
+/etc/init.d/S83endpoint_adapter_rpmsg_piksi100 stop
```
Comment (silverjam) [inline package/piksi_system_daemon/overlay/etc/init.d/update_ntp_config]: Why are we killing the rpmsg adapter, but not restarting it?

Comment (benjaminaltieri) [reply package/piksi_system_daemon/overlay/etc/init.d/update_ntp_config]: Good catch, copy/paste error - fixed!

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v1.5.0-develop-2018072722-8-gfe605f21
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=607284)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.5.0-develop-2018072722-8-gfe605f21&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.5.0-develop-2018072722-8-gfe605f21&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v1.5.0-develop-2018072722-8-gfe605f21
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v1.5.0-develop-2018072722-8-gfe605f21)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.5.0-develop-2018072722-8-gfe605f21/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.5.0-develop-2018072722-8-gfe605f21

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v1.5.0-develop-2018072722-8-g97166e15
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=607497)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.5.0-develop-2018072722-8-g97166e15&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.5.0-develop-2018072722-8-g97166e15&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v1.5.0-develop-2018072722-8-g97166e15
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v1.5.0-develop-2018072722-8-g97166e15)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.5.0-develop-2018072722-8-g97166e15/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.5.0-develop-2018072722-8-g97166e15

**Pull request merged**